databases/week2/school-db.sql

```diff
@@ -0,0 +1,37 @@
+USE `school-db`;
+
+CREATE TABLE `class` (
+  `id` int unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) NOT NULL,
+  `begin` DATETIME,
+  `ends`DATETIME NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+
+CREATE TABLE `student` (
+`id` int unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) NOT NULL,
+  `email` varchar(255) NOT NULL,
+  `phone` varchar(255) NULL,
+  `class_id` int unsigned NOT NULL,
+  PRIMARY KEY (`id`)
+  -- CONSTRAINT `fk_class` FOREIGN KEY `class`(id) REFERENCES `student`(id) ON UPDATE CASCADE ON DELETE CASCADE
+  ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+
+insert into class (id, name, begin, ends, status) values (1, 'Math', '2017-10-25', '2019-10-15', 2);
```

Comment:
Usually it's a bad idea to insert the primary key (id in this case) manually, the system does it automatically when you create the column with "AUTO_INCREMENT", and that way we never get errors because we manually inserted a value that already exists.

Also there is no column named status in the class table, so this query and the next two just give an error.

Reply:
yes true, I edit it, and it works as you said. So I removed the id as an id is auto increment.